### ISSUE TITLE
Create Maemo Libre (deblobbed) images

### ISSUE THREAD

**Issue by dderby** [maemo-leste/bugtracker#243]
Briefly discussed [here](https://talk.maemo.org/showpost.php?p=1541043&postcount=35), it would be nice to be able to offer a deblobbed version of Maemo Leste.

**Comment by dderby** [comment]
Referencing issue #51.  We'll want this for any devices that need blobs for Hildon to be usable (e.g. PVR devices).

**Comment by MerlijnWajer** [comment]
I don't see #51 happening any time soon, so I wouldn't count on it, unfortunately.

**Comment by eloydegen** [comment]
Is the image for Allwinner with Mali graphics already a deblobbed one? If it isn't, I think that would be the most feasible candidate thanks to Lima.

**Comment by dderby** [comment]
@eloydegen, in the case of the PinePhone, no.  Blobs are required for Wi-Fi, Bluetooth and Camera auto-focus.  That said, given that #51 won't be happening anytime soon, that would make the PinePhone would be the perfect candidate for a deblobbed version of Maemo.

**Comment by dderby** [comment]
I didn't make this clear in the first post, but the idea behind this is to create a GNU/Linux alternative to Replicant (for which there are currently no alternatives at all).  It won't be too difficult to do.  Wi-Fi, Bluetooth, etc. don't need to work.  You could even connect a blob-free RYF USB dongle if you want Wi-Fi.  We could probably get it working with Linux-libre kernel too, which would make it superior to Replicant from a free software perspective and may lead to FSF endorsement.

**Comment by MerlijnWajer** [comment]
Right, you could plug in hardware that also requires firmware, but instead has it burned into the device itself... And that somehow makes it 'free' as in freedom...
